### ISSUE TITLE
many behavoirs are not documented

### ISSUE THREAD

**Issue by pdurbin** [publiclab/plotsbot#142]
Over at https://gitter.im/publiclab/publiclab?at=5c0d471df992693c7a4913e2 I hear there are many undocumented behaviors. We should add them to the README.

**Comment by jywarren** [comment]
indeed! could look in /src/behaviors, which is reasonably well commented

**Comment by ryzokuken** [comment]
@jywarren do you want me to make FTOs for each behavior? I feel it should be pretty straightforward to just look at all the .js files.

**Comment by jywarren** [comment]
sure, but perhaps the "denoisy-fying" issue for the issue I just opened is
a higher priority to start with if you have a moment! Thanks!
